tests/test_generic.py

```diff
@@ -972,32 +972,32 @@ def test_create_string_object_force():
 @pytest.mark.parametrize(
     ("value", "expected"),
     [
-        ("0.000000", "0"),
-        ("0.0", "0"),
+        ("0.000000", "0.0"),
+        ("0.0", "0.0"),
         ("1.0", "1"),
         ("0.123000", "0.123"),
         ("0.000123000", "0.000123"),
-        ("0.0", "0"),
-        ("0", "0"),
+        ("0.0", "0.0"),
+        ("0", "0.0"),
         ("1", "1"),
         ("1.0", "1"),
         ("1.01", "1.01"),
         ("1.010", "1.01"),
-        ("0000.0000", "0"),
+        ("0000.0000", "0.0"),
         ("0.10101010", "0.1010101"),
         ("50000000000", "50000000000"),
-        ("99900000000000000123", "99900000000000000123"),
-        ("99900000000000000123.456000", "99900000000000000123.456"),
+        ("99900000000000000123", "99900000000000000000"),
+        ("99900000000000000123.456000", "99900000000000000000"),
         ("0.00000000000000000000123", "0.00000000000000000000123"),
         ("0.00000000000000000000123000", "0.00000000000000000000123"),
-        (
-            "50032481330523882508234.00000000000000000000123000",
-            "50032481330523882508234.00000000000000000000123",
-        ),
-        (
-            "928457298572093487502198745102973402987412908743.75249875981374981237498213740000",
-            "928457298572093487502198745102973402987412908743.7524987598137498123749821374",
-        ),
+        # (
+        #    "50032481330523882508234.00000000000000000000123000",
+        #    "50032481330523882508234.00000000000000000000123",
+        # ),
+        # (
+        #    "928457298572093487502198745102973402987412908743.75249875981374981237498213740000",
+        #    "928457298572093487502198745102973402987412908743.7524987598137498123749821374",
+        # ),
```

Comment:
Why comment these test cases out vs fully delete them?

Reply:
Just wondering if it worth to upgrade the test for those numbers too. your opinion ?

Reply:
I suppose it'll be something could point at how precision will be lost on plugging in these ridiculous numbers into a `FloatObject`. :shrug: